python/py-dmgbuild/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-dmgbuild
+version             1.6.2
+
+categories-append   devel
+license             {MIT PSF}
+platforms           {darwin any}
+supported_archs     noarch
+maintainers         nomaintainer
+description         macOS command line utility to build disk images
+long_description \
+    dmgbuild is a command line tool to create macOS disk images (aka .dmg files). While it is possible to create disk images easily enough from the command line using the hdiutil program that ships with macOS, there is no easy way to configure the appearance of the resulting disk image when the user opens it.
+
+homepage            https://dmgbuild.readthedocs.io/en/latest/
+
+checksums           md5     cc46f3a49ebbd014c83b6dd6faa16377 \
```

Comment:
"md5" is obsolete

Reply:
Weird. I thought I did it with the value returned by `port` itself. Maybe I based myself off an old portfile or something like that?